{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -174,7 +174,7 @@ SITE_API_URL = "https://127.0.0.1/api"
 
 # Invenio-RDM-Records
 # ===================
-# See https://inveniordm.docs.cern.ch/customize/dois/
+# See https://inveniordm.docs.cern.ch/operate/customize/dois/#parent-or-concept-dois
```

Comment:
I think you want https://inveniordm.docs.cern.ch/operate/customize/dois/, not the parent subsection